applications/StructuralMechanicsApplication/tests/test_patch_test_large_strain.py

```diff
@@ -419,8 +443,15 @@ def test_TL_2D_triangle(self):
 
         A,b = self._define_movement(dim)
 
+        self._ResetDisplacementAndPosition(mp)
+        self._apply_BCs(bcs,A,b)
+        self._solve(mp,"block_builder")
+        self._check_results(mp,A,b)
```

Comment:
note that this check fails even if you simply call the _solve function twice, independently on which builder and solver is used.